--- configserver/src/main/java/io/steeltoe/docker/configserver/ConfigServer.java	2024-02-21 13:33:04.000000000 -0600
+++ configserver/src/main/java/io/steeltoe/docker/configserver/ConfigServer.java	2024-02-21 13:40:40.622446300 -0600
@@ -1,12 +1,21 @@
 package io.steeltoe.docker.configserver;

+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.config.server.EnableConfigServer;

 @SpringBootApplication
+@EnableConfigServer
 public class ConfigServer {

+	private static final Logger logger = LoggerFactory.getLogger(ConfigServer.class);
+
 	public static void main(String[] args) {
+        System.setProperty("spring.config.name", "configserver");
+        Package pkg = EnableConfigServer.class.getPackage();
+        logger.info("{} {} by {}", pkg.getImplementationTitle(), pkg.getImplementationVersion(), pkg.getImplementationVendor());
 		SpringApplication.run(ConfigServer.class, args);
 	}

